src/todo/UI.lua

```diff
@@ -222,16 +246,23 @@ function todo.create_add_edit_frame(player, task)
             name = "todo_persist_button",
             caption = {"todo.persist"}
         })
-        flow.add({
-            type = "checkbox",
-            style = "todo_checkbox_default",
-            name = "todo_add_top",
-            state = false,
-            caption = {"todo.add_top"}
-        })
     end
 end
 
+function todo.create_delete_confirmation_button(element, id)
+    local table = element.parent
+    todo.log(table)
```

Comment:
This log is not really understandable when in the game. Could you add a log before describing what is printed?

Reply:
This is actually a debug output. I'll simply remove it.